Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -546,13 +547,59 @@ private void DumpLists(BuildResult actualResult, string itemType, string[] expec
 
             TestContext.WriteLine("");
             TestContext.WriteLine("Dumping <" + itemType + "> list: actual");
-            foreach (var item in actualResult.ProjectStateAfterBuild.GetItems(itemType))
+            foreach (var item in actualResult.GetCapturedItemValues(itemType))
             {
-                TestContext.WriteLine("\t{0}", item.EvaluatedInclude);
+                TestContext.WriteLine("\t{0}", item);
             }
             TestContext.WriteLine("");
         }
 
+        /// <summary>
```

Comment:
Added some equivalent assertion methods that check the content of the build log.

The assertion methods that these replace were in the shared _BuildAssertions_ file. However, this is the only test class that used them so I put the replacement methods here.